server/game/game.js

```diff
@@ -113,6 +121,47 @@ class Game extends EventEmitter {
         this.attackState = null;
     }
 
+    calculateExpectedResults() {
```

Comment:
I don't think this belongs in Game. put it in a helper class and pass in the bare minimum needed for in-game display (I assume)

Reply:
It's not for display purposes. Expected value is a value between 0-1 that indicates the probability a given player wins. That plus the actual outcome is then fed into the formula for updating the rating. It's on a per game basis as it is based on each player's current rating. I put it in Game as I wasn't sure how it would behave on rematch or disconnects; I don't want it lingering between a player's separate games. Making it a method in the EloCalculator should be fine.